applications/minotari_merge_mining_proxy/src/cli.rs

```diff
@@ -30,6 +30,8 @@ use tari_common::configuration::{ConfigOverrideProvider, Network};
 pub struct Cli {
     #[clap(flatten)]
     pub common: CommonCliArgs,
+    #[clap(short, long, alias = "non-interactive", env = "TARI_NON_INTERACTIVE")]
```

Comment:
```suggestion
    #[clap(short, long, alias = "non-interactive", env = "MINOTARI_NON_INTERACTIVE")]
```

Reply:
There is a good chance this will be used over both base layer apps and dan, prefer to keep it as is for now